### PR TITLE
Initial release of n-quote

### PR DESCRIPTION
**Pull request by tom-parker**

### CODE REVIEW

**Comment by keirog** [comment]
Nice genre selectors :+1:

**Comment by keirog** [comment]
Probably a good idea to match [o-quote](https://github.com/Financial-Times/o-quote)'s conventions at some point

File: README.md [hunk 1]
```diff
@@ -1 +1,40 @@
...
+`@import 'n-quote/main';`
+
+####HTML:
+
+```
+<blockquote class="n-quote">
+	<p>Lorem ipsum...</p>
+</blockquote>
+```
+```
```
Comment (kaelig) [inline README.md]: Adding `html` at the beginning of the code block will enable syntax highlighting

File: demos/main.html [hunk 2]
```diff
@@ -0,0 +1,35 @@
...
+				<p>These budget cuts are absolutely unprecedented — its gobsmacking</p>
+				<cite class="n-quote__cite">Helen Szoke, Oxfam Australia chief executive</cite>
+			</blockquote>
+		</div>
+	</div>
+	<div class="o-grid-row">
+		<p class="variant-header">Comment pull quote</p>
+	</div>
+	<div class="o-grid-row">
+		<div data-o-grid-colspan="12" class="article--comment">
```
Comment (kaelig) [inline demos/main.html]: Can a block that doesn't have a block class be using a modifier?

As in: should this be `article article--comment`?

File: demos/main.html [hunk 3]
```diff
@@ -0,0 +1,35 @@
+<div class="o-grid-row">
+	<div class="o-grid-row">
```
Comment (kaelig) [inline demos/main.html]: Rows can't be direct children of rows

File: demos/main.html (hunk 2 above)
Comment (tom-parker) [inline demos/main.html]: Very true, I'll update the demo.

File: demos/main.html [hunk 4]
```diff
@@ -0,0 +1,33 @@
+<div class="o-grid-row">
+	<p class="variant-header">Default pull quote</p>
```
Comment (kaelig) [inline demos/main.html]: Rows usually contain a column inside (unless `.variant-header` already has column styles?)

File: demos/main.html [hunk 5]
```diff
@@ -0,0 +1,33 @@
...
+<div class="o-grid-row">
+	<div data-o-grid-colspan="12" class="article article--comment">
+		<blockquote class="n-quote">
+			<p>These budget cuts are absolutely unprecedented — its gobsmacking</p>
+			<cite class="n-quote__cite">Helen Szoke, Oxfam Australia chief executive</cite>
+		</blockquote>
+	</div>
+</div>
+<div class="o-grid-row">
+	<p class="variant-header">Lifestyle pull quote</p>
```
Comment (kaelig) [inline demos/main.html]: Same here

**Pull request merged**